test/helpers/utils.go

```diff
@@ -128,6 +128,23 @@ func (c *TimeoutConfig) Validate() error {
 // the timeout in config is reached. Returns an error if the timeout is
 // exceeded for body to execute successfully.
 func WithTimeout(body func() bool, msg string, config *TimeoutConfig) error {
+	err := RepeatUntilTrue(body, config)
```

Comment:
any reason why we don't just `return RepeatUntilTrue(body, config)` here?

Reply:
Because it would not print the `msg` as before